front_end/panels/ai_chat/ui/SettingsDialog.ts

```
@@ -2319,6 +2421,459 @@ export class SettingsDialog {
+
+    const mcpStatusText = document.createElement('span');
+    mcpStatusText.className = 'connection-status-text';
+    mcpStatusContainer.appendChild(mcpStatusText);
+
+    const mcpStatusDetails = document.createElement('div');
+    mcpStatusDetails.className = 'settings-hint';
+    mcpStatusDetails.style.marginTop = '4px';
+    mcpSection.appendChild(mcpStatusDetails);
+
+    const formatTimestamp = (date: Date | undefined): string => {
+      if (!date) return '';
```

ESLint

[error] Expected { after 'if' condition.

(curly)